app/src/main/java/diemmy/edu/recycler/ItemView.java

```diff
@@ -0,0 +1,29 @@
+package diemmy.edu.recycler;
+
+/**
+ * Created by DELL on 10/10/2017.
+ */
+
+public class ItemView {
+    private String name;
+    private String gia;
```

Comment:
em nên sử dụng đồng nhất 1 cách đặt tên biến, class, string. K nên đặt nửa tây nửa ta, nửa nạc nửa mỡ ntn.
suggest change gia to price